### ISSUE TITLE
Suggestion: pivot to "public-domain-affirmation"

### ISSUE THREAD

**Issue by breck7** [good-labs/greater-good-affirmation#29]
I found this site via Tauri. Interesting concept. Have you considered a slight tweak and calling it "public-domain-affirmation"? I look at things like tcp/ip, www, and more recently sqlite, and notice that the things being done for "the greater good", are all just put into the public domain. Perhaps you could kick off a trend where people loudly and proudly declare their support for public domain?

Just throwing this out there. Feel free to close. Thought sometimes a little tweak can make all the difference.

**Comment by vsoch** [comment]
That's an interesting idea! So the affirmation would be to put some piece of work or project in the public domain, meaning a specific set of [licenses](https://en.wikipedia.org/wiki/Public-domain-equivalent_license)? That's an interesting idea, but if it's contingent on a license, I have a lot of work that I would consider "for the greater good" that doesn't have any of those particular licenses. But I really like that idea - do you think there should be more code in the public domain?

**Comment by breck7** [comment]
Yes, the affirmation would be just to put it in the public domain. You really can put something in the public domain by just clearly writing "public domain". No "license" is needed. Which is why your badge idea could be really neat. People could see it and know "oh, this is public domain", and then click it and hit your page where they could find other public domain software. You would blaze new ground because legally you don't need to have a "license", and adding your badge would be enough, and allow passionate public domain people (sqliters, or /r/publicdomain,for example) to discover other public domain software.

Yes, I would LOVE to see more public domain software.

**Comment by vsoch** [comment]
That's a good point - so you've pointed out that it's valuable to have public domain software, but maybe there isn't enough appreciation or accolade for it. An interesting idea (although I'm not sure how feasible) would be to make an equivalent repo like this one, but for the `public-domain-affirmation`, and then have some kind of bot that can (based on the license) automatically open PRs to give a badge. The question, however, is why having a badge is useful, or would be wanted. I could see the PRs as just being an annoyance.

**Comment by breck7** [comment]
Fantastic idea. Maybe will experiment with something along those lines at some point in time.

**Comment by vsoch** [comment]
If you'd like to join the good labs, you are welcome to create a repository here! I can help if you want it. It's a cool idea - I think the hard part is less building the tool / interface, moreso convincing people of the importance and to embrace it.

**Comment by breck7** [comment]
Happy to give it a go!

**Comment by vsoch** [comment]
Awesome! Ping me for points of discussion, or other kind of technical help. Looking forward to it!
